content/shmem_fence.tex

```diff
@@ -19,18 +16,14 @@
 \end{apiarguments}
 
 \apidescription{
-    This routine assures ordering of delivery of memory store, blocking \PUT{},
-    \ac{AMO}, and \OPR{put-with-signal}, as well as nonblocking \PUT{},
-    \OPR{put-with-signal}, and \ac{AMO}
-    routines to symmetric data objects.  All memory store, blocking \PUT{},
-    \ac{AMO}, and \OPR{put-with-signal}, as well as nonblocking \PUT{},
-    \OPR{put-with-signal}, and \ac{AMO}
-    routines to symmetric data objects issued to a particular remote \ac{PE}
-    on the given context prior
+    This routine assures ordering of delivery of operations to symmetric data
+    objects.
+    Refer Table~\ref{mem-order} for the list of operations that are ordered
+    by \FUNC{shmem\_fence} routine.
+    All operations to symmetric data objects issued to a particular
+    remote \ac{PE} on the given context prior
```

Comment:
Do we really want to keep "remote" here, since `shmem_fence` also orders operations operating on the calling PE?

Reply:
I think we should remove ```remote``` from here. 

Also, in general, do we need to say something about the operation of the ```shmem_ctx_fence``` with respect to ```SHMEM_CTX_NOSTORE```. Reading the NOSTORE, it kind of confuses how it related to the stores in this description.

Reply:
I'm neutral on whether to handle `SHMEM_CTX_NOSTORE` here. The description of the NOSTORE option in `shmem_ctx_create` creates an exception to the behaviors listed here in `shmem_fence`. That seems sufficient to me, but I'd also be okay if something was added here since the potential for overly duplicating text is limited to fence and quiet.